layers/+lang/javascript/packages.el

```diff
@@ -65,6 +66,21 @@
   ;; Required to make imenu functions work correctly
   (add-hook 'js2-mode-hook 'js2-imenu-extras-mode))
 
+(defun javascript/init-npm-mode ()
+  (use-package npm-mode
+    :defer nil
+    :init
+    (progn
+      (add-hook 'js2-mode-hook #'npm-mode))
+    :config
+    (progn
+      (spacemacs/declare-prefix-for-mode 'js2-mode "mn" "npm")
+      (spacemacs/set-leader-keys-for-major-mode 'js2-mode
+        "ni" 'npm-mode-npm-install
```

Comment:
How about adding other commands and keybindings?

https://github.com/mojochao/npm-mode/blob/3ee7c0bad5b7a041d4739ef3aaa06a3dc764e5eb/npm-mode.el#L34-L41

Reply:
Please also add the following missing commands for a full integration:
Command|Binding|Description
--------------|----------|---------------
npm-mode-npm-init             | <kbd>SPC m n n</kbd> | Initialize new project
npm-mode-npm-uninstall        | <kbd>SPC m n u</kbd> | Remove project dependency           
npm-mode-npm-list             | <kbd>SPC m n l</kbd> | List installed project dependencies 
npm-mode-visit-project-file   | <kbd>SPC m n p</kbd> | Visit project package.json file

Reply:
Added commands as requested.  Thanks for the suggestion!